lib/compressor/restructure/4-restructShorthand.js

```diff
@@ -118,6 +118,16 @@ TRBL.prototype.getValueSequence = function(value, count) {
             case 'Percentage':
                 break;
 
+            case 'Function':
+                switch (child.name) {
```

Comment:
may appear new rules to other function therefore i added switch construction without if construction